src/NuGetSdkResolver/NuGetSdkResolver.cs

```diff
@@ -203,8 +203,15 @@ private static bool TryGetMSBuildSdkPackageInfo(FallbackPackagePathResolver fall
                     return false;
                 }
 
-                // Get the installed path and add the expected "Sdk" folder
+                // Get the installed path and add the expected "Sdk" folder.  Windows file systems are not case sensitive
```

Comment:
If `sdk` is the expected/documented casing as mentioned in https://github.com/Microsoft/msbuild/issues/2803#issuecomment-360624653 I think we should flip the check around?